src/CustomerProfile.js

```diff
@@ -0,0 +1,96 @@
+import React, { useState, useEffect } from "react";
+
+const CustomerProfile = (props) => {
+  const [customerProfileData, setCustomerProfileData] = useState(null);
+  const [isLoading, setIsLoading] = useState(true);
+
+  useEffect(() => {
+    setIsLoading(true);
+    fetch(`https://cyf-react.glitch.me/customers/${props.id}`)
```

Comment:
In Bookings.js you use async, await to do the fetch, while here you use .then() (i.e. promises chaining). I suggest using async, await here as well to make sure the code is consistent.